api/v1/requests/AttendeeRequest.js

```diff
@@ -40,6 +40,10 @@ var bodyValidations = {
 	'attendee.linkedin':  ['required', 'string', 'maxLength:50'],
 	'attendee.interests': ['required', 'string', 'maxLength:255'],
 	'attendee.status': ['string', registration.verifyStatus],
+	'attendee.priority': ['required', 'integer', 'max:10', 'min:0'],
```

Comment:
It looks like reviewers have to send up all registration information just to mark someone as accepted or not. Do you think there should be a separate request for this? Also, is there anywhere where you prevent the non-organizer users from deciding their own priority, wave, etc?